### DISCUSSION TITLE
What's the projectId to run the samples/wallet app?

### DISCUSSION THREAD

**Discussion by willyfu** [WalletConnect/walletconnect-monorepo#2592] [Developer Support]
What's the projectId to run the samples/wallet app? Thanks!

**Comment by finessevanes** [comment]
You'll have to create a new project ID. It's really easy: https://docs.walletconnect.com/2.0/cloud/explorer#setting-up-a-new-project

**Comment by dhorkin** [comment]
> You'll have to create a new project ID. It's really easy: https://docs.walletconnect.com/2.0/cloud/explorer#setting-up-a-new-project

That does not answer the question. He asked _what is_ not _where do I get one_.

I am also in a similar boat. I was considering using `Web3Modal` in my React project (which is using `wagmi`), but there is **zero** explanation for why the `projectId` parameter is required as opposed to optional.

I don't see a reason to register with an external service for an ID just to get a nice UI element. I like the sample on the main [Web3Modal ](https://web3modal.com/) page, but unless an explanation is needed for why I need cloud services to connect to a wallet... I'm going to be passing on this library.

This [page](https://docs.walletconnect.com/2.0/cloud/explorer#setting-up-a-new-project) says
> The WalletConnect Cloud [Explorer](https://walletconnect.com/explorer) is an open-source solution to submit and showcase wallets and dApps that support WalletConnect. You can view existing entries over at our Explorer or use the [Cloud Explorer API](https://docs.walletconnect.com/2.0/cloud/explorer#cloud-explorer-api) to fetch them programmatically.

So, it sounds like projectId is **exclusively** used to publish your app in a Walletconnect registry, which is something I *do not want to do* for my project, which is *not* public facing.

Or perhaps it has something to do with the relay, but again, why do I need a relay?
